testing/ppsspp/APKBUILD

```diff
@@ -0,0 +1,82 @@
+# Contributor: Díaz Urbaneja Diego <sodomon2@gmail.com>
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=ppsspp
+pkgver=1.8.0
+pkgrel=0
+pkgdesc="PPSSPP - a fast and portable PSP emulator"
+url="https://www.ppsspp.org/"
+arch="x86_64"
+license="GPL-2.0-only"
+depends="cmake make"
+makedepends="glew-dev libzip-dev snappy-dev mesa-dev qt5-qtbase-dev zlib-dev sdl2-dev"
+options="!check" # make check not implemented
+source="$pkgname-$pkgver.tar.gz::https://github.com/hrydgard/$pkgname/archive/v$pkgver.tar.gz
+  $pkgname-$pkgver-armips.zip::https://github.com/Kingcom/armips/archive/7885552b208493a6a0f21663770c446c3ba65576.zip
+  $pkgname-$pkgver-discord-rpc.zip::https://github.com/discordapp/discord-rpc/archive/3d3ae7129d17643bc706da0a2eea85aafd10ab3a.zip
+  $pkgname-$pkgver-glslang.zip::https://github.com/hrydgard/glslang/archive/f9d08a25fbe17e0677a89d398f4d7f232339c3f9.zip
+  $pkgname-$pkgver-SPIRV-Cross.zip::https://github.com/KhronosGroup/SPIRV-Cross/archive/be7425ef70231ab82930331959ab487d605d0482.zip
+  $pkgname-$pkgver-rapidjson.zip::https://github.com/Tencent/rapidjson/archive/73063f5002612c6bf64fe24f851cd5cc0d83eef9.zip
+  $pkgname-$pkgver-lang.zip::https://github.com/hrydgard/$pkgname-lang/archive/03707d64bffc594d5707780a34ada77e8f43cdc0.zip
+  $pkgname-$pkgver-ffmpeg.zip::https://github.com/hrydgard/$pkgname-ffmpeg/archive/7472c903ec771c00af9925097e8d37075e9a379f.zip
+  ppsspp.desktop
+  ppsspp
+  ppsspp-version.patch
+  ppsspp-headless"
+
+prepare(){
+  cd "$srcdir/"
+  #adding submodules needed by ppsspp
+  
+  mv armips-7885552b208493a6a0f21663770c446c3ba65576/* "$builddir"/ext/armips/
+  mv discord-rpc-3d3ae7129d17643bc706da0a2eea85aafd10ab3a/* "$builddir"/ext/discord-rpc/
+  mv glslang-f9d08a25fbe17e0677a89d398f4d7f232339c3f9/* "$builddir"/ext/glslang/
+  mv rapidjson-73063f5002612c6bf64fe24f851cd5cc0d83eef9/* "$builddir"/ext/rapidjson/
+  mv SPIRV-Cross-be7425ef70231ab82930331959ab487d605d0482/* "$builddir"/ext/SPIRV-Cross/
+  mv $pkgname-lang-03707d64bffc594d5707780a34ada77e8f43cdc0/* "$builddir"/assets/lang/
+  mv $pkgname-ffmpeg-master/* "$builddir"/ffmpeg
+  cd "$builddir"/ffmpeg
+  ./linux_x86-64.sh
+}
+
+build() {
+  patch git-version.cmake < $srcdir/ppsspp-version.patch
```

Comment:
Why not add this to `source` so abuild does this for you?

Reply:
as I would since I've tried but it doesn't work.

Reply:
For some reason I don't apply the patch alone, and that's why I put it like this.

Reply:
It's probably not applied because you're not running `default_prepare` in `prepare()`.